connector/connect/client/jvm/src/main/scala/org/apache/spark/sql/Column.scala

```diff
@@ -1312,6 +1312,10 @@ private[sql] object Column {
     new Column(builder.build())
   }
 
+  def apply(extension: com.google.protobuf.Any): Column = {
+    apply { builder => builder.setExtension(extension) }
```

Comment:
does this pass style checks?

OCD: this also works: `apply(_.setExtension(extension))`